tracing-attributes/src/lib.rs

```diff
@@ -294,6 +305,18 @@ mod expand;
 ///     Ok(())
 /// }
 /// ```
+/// 
+/// If you're returning a [`std::result::Result<T, E>`] and `E` implements [`std::error::Error`], it can be
+/// recorded the raw error value using `err(Value)`. This can be useful when paired with a subscriber
+/// which specially handles errors, like [`tracing_opentelemetry`]:
```

Comment:
docs nit: we are specifically referring to the `Error` trait when we say "specially handles errors", here, so I might make that a little clearer:
```suggestion
/// which specially handles `Error`s, like [`tracing_opentelemetry`]:
```